src/routes/homepage-redesign-preview/index.svelte

```diff
@@ -1,106 +1,260 @@
+    <Button icon="{ArrowRight16}" href="/about" sapper:prefetch
+      >More about Cal-Adapt</Button
+    >
+  </div> -->
+  <div class="cta-card" slot="cta-card">
+    <Card
+      {...{
+        titleText: "Cal-Adapt is evolving!",
+        linkPath: `/help/`,
+        description:
+          "Learn about the Cal-Adapt enterprise and our mission to support California's climate change initiatives and preview our future plans.",
+        height: 18,
+        ctaText: "Read More",
+      }}
+    >
+      <div class="icon-circle bg-teal-60" slot="icon_slot">
+        <svelte:component this="{InformationFilled32}" fill="red" />
+      </div>
+    </Card>
+  </div>
+</Banner>
 
 <div class="spacing--v-32"></div>
 
-<Grid>
-  <Row>
-    <Column lg="{{ span: 12 }}">
+<div class="bx--grid">
+  <div class="bx--row">
+    <div class="bx--col-lg-12 bx--col-md-8 bx--col-sm-4">
+      <!-- Fifth Assessment Notification -->
+      <div class="alert">
+        <InlineNotification
+          lowContrast
+          hideCloseButton
+          kind="info"
+          title="Looking for climate data for California's Fifth Climate Change Assessment?"
+        >
+          <p class="bx--inline-notification__subtitle">
+            Visit the <a href="/data">Cal-Adapt Analytics Engine</a>
+          </p>
+        </InlineNotification>
+      </div>
+
+      <!-- Fourth Assessment Block -->
+      <h2>Explore interactive maps and charts</h2>
+      <div class="spacing--v-24"></div>
+      <p class="full-width">
+        Visualize and download <strong>downscaled CMIP5 climate data</strong>
+        and other datasets developed for California’s
+        <a href="/blog/slr-cis-tool" target="_blank"
+          >Fourth Climate Change Assessment.</a
+        >
+        Read our <a href="/get-started" target="_blank">Get Started</a> guide to
+        learn more about working with climate data.
+      </p>
+      <p class="full-width">
+        Designed for a <strong>broad range of users.</strong>
+      </p>
+      <div class="spacing--v-24"></div>
       <CardsContainer gridGap="{2}" cardWidth="{cardWidth}">
-        {#each cardsData as card}
+        {#each cardsData as cardDatum, index}
           <Card
             {...{
-              ...card,
+              ...cardDatum,
               height: cardHeight,
+              ctaText: "Learn More",
+              textColor: "white",
               useRule: true,
-              textColor: "var(--white)",
             }}
           />
         {/each}
       </CardsContainer>
-    </Column>
 
-    <Column lg="{{ span: 3, offset: 1 }}">
-      <h2 class="h4">Latest Updates</h2>
-      <p>
-        Learn more about the latest changes and improvements to Cal-Adapt on our
-        blog.
+      <!-- Analytics / Fifth Assessment Block -->
+      <div class="spacing--v-24"></div>
+      <h2>Analyze next generation climate data</h2>
+      <div class="spacing--v-24"></div>
+
+      <div class="bx--grid">
```

Comment:
To make content within the nested grid line up with parent grid elements here you can add negative margins to the nested grid. Try adding `margin-right: -1rem; margin-left: -1rem;`

Reply:
I tried but still not quite there :( 
<img width="593" alt="Screen Shot 2023-02-28 at 13 03 35" src="https://user-images.githubusercontent.com/124095974/221979186-483fa7a8-e73c-48e1-a69e-1739ccde57ee.png">

Reply:
Should I apply this CSS to every `bx--grid` or nested grid, or just the one in this page?

Reply:
The Fourth Assesment Cards are laid out using a `ul` element with the following css:

```
ul.svelte-hbb6rw {
  width: 100%;
  display: grid;
  grid-template-columns: repeat( auto-fill, minmax(var(--card-width, 16rem), 1fr) );
  grid-gap: var(--gap, 1rem);
  list-style: none;
  padding: 0;
  margin: 0;
}
```

If you use a similar CSS grid class to layout the AE blocks (using 2 blocks instead of the 3 above) I don't think you will need the nested `bx--grid`, `bx--row` divs at all. So you won't need to add any negative margins. Something like:
```
div {
  width: 100%;
  display: grid;
  grid-template-columns: 18rem auto;
  grid-gap: 2rem;
}
```

Reply:
Not sure why, but a value of `22rem` works better for making the AE image block identical to the cards instead of `18rem` in my example above. Probably something to do with `auto-fill` prop.

Reply:
@mukhtyar I've implemented the grid now. `22rem` indeed approximates the size of the cards on desktop. However, it breaks down when looking at it with smaller screens. See the image for an example. Should we just ignore this, or dig deeper? 
![Screen Shot 2023-03-04 at 23 12 39](https://user-images.githubusercontent.com/124095974/222947078-bac958f1-171f-4b53-bd32-59ccc90af919.png)